### DISCUSSION TITLE
Bot does not work in a group

### DISCUSSION THREAD

**Discussion by vyacheslav25** [metalmatze/alertmanager-bot#142] [Help/Support]
Everything works with a personal id.
If you insert the group id, then the bot does not show signs of life

`level=info ts=2021-01-31T16:14:16.17329793Z caller=bot.go:252 component=telegram msg="failed to process message" err="dropped message from forbidden sender" sender_id=3333333 sender_username=user123`

Group ID looked with @RawDataBot.
-571999999(example)

Bot /setprivacy DISABLED

Help, please

**Comment by fajfer** [comment]
Can't replicate that, works in a group for me

**Comment by vyacheslav25** [comment]
> Can't replicate that, works in a group for me

Did you use base64 for encoding?

**Comment by vyacheslav25** [comment]
Sorry, I found that you cannot use the group ID
